consensus/votepower/roster.go

```diff
@@ -154,58 +154,58 @@ func (r *Roster) JSON() string {
 		Voters map[string]stakedVoter `json:"voters"`
 		Our    string                 `json:"ours"`
 		Their  string                 `json:"theirs"`
-		Raw    string                 `json:"raw-total"`
+		Raw    string                 `json:"total-effective-stake"`
 	}{
 		v,
 		r.OurVotingPowerTotalPercentage.String(),
 		r.TheirVotingPowerTotalPercentage.String(),
-		r.RawStakedTotal.String(),
+		r.TotalEffectiveStake.String(),
 	}
 	b, _ := json.Marshal(&c)
 	return string(b)
 }
 
 // Compute creates a new roster based off the shard.SlotList
+// TODO(audit): the decimal computation is memory and cpu intensive.
```

Comment:
This compute happens once

Reply:
it happens in accumulate rewards every block